Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM golang:alpine3.15
-
-WORKDIR /root/
-COPY ./go.mod ./go.sum ./
+FROM golang as build
+WORKDIR /app
+COPY . .
 RUN go mod download
-COPY ./lazytainer.go .
-RUN go build lazytainer.go
-
-FROM alpine:latest
-WORKDIR /root/
-COPY --from=0 /root/lazytainer ./
+RUN go build -trimpath -ldflags="-s -w" lazytainer.go
 
-CMD ["./lazytainer"]
\ No newline at end of file
+FROM scratch
```

Comment:
I'm fine if you create another PR with all of your non-functional changes, as I think they look good. But I think the scratch image is fundamentally incompatible with lazytainer as it is now. The netstat portion requires a shell to exist in order to run. Did you try running this before you created this PR?

Reply:
Yes it does require shell, I've tested before pushing it but looks like for some caching reason it didn't used `scratch` one and used alpine one that I was testing before it. Made changes, Will push it 

https://github.com/vmorganp/Lazytainer/pull/6#discussion_r962092021 regarding this, Since you're trying to stay in alpine side, Let me install g++ in golang alpine.

FYI it makes 4MB difference (non-compressed)